internal/consts/consts.go

```diff
@@ -284,6 +284,18 @@ const (
 	FieldIssuerID                      = "issuer_id"
 	FieldKeyRef                        = "key_ref"
 	FieldPemBundle                     = "pem_bundle"
+	FieldCAChain                       = "ca_chain"
```

Comment:
This small switch must have happened during all the merge conflict resolutions 😅 Happy to move it back if folks really feel the need for it but left it as is for now